### PR TITLE
2 configuration flags, --peer-ca-file and --ca-file were  removed

### PR DESCRIPTION
**Pull request by Somoshree**
closes: #422

### CODE REVIEW

**Comment by nate-double-u** [comment]
Deploy preview: https://deploy-preview-473--etcd.netlify.app/docs/v3.5/op-guide/configuration/#security-flags

File: content/en/docs/v3.5/op-guide/configuration.md [hunk 1]
```diff
@@ -248,11 +248,9 @@ The security flags help to [build a secure etcd cluster][security].
 
 ### --ca-file
 
-**DEPRECATED**
+**REMOVED**
 
-+ Path to the client server TLS CA file. `--ca-file ca.crt` could be replaced by `--trusted-ca-file ca.crt --client-cert-auth` and etcd will perform the same.
-+ default: ""
-+ env variable: ETCD_CA_FILE
+This flag has been removed. `--trusted-ca-file ca.crt --client-cert-auth` can be used to perform the same task as was performed by `--ca-file ca.crt` previously.
```
Comment (nate-double-u) [inline content/en/docs/v3.5/op-guide/configuration.md:253]:
```suggestion
+ This flag has been removed. `--trusted-ca-file ca.crt --client-cert-auth` can be used to perform the same task as was performed by `--ca-file ca.crt` previously.
```
Let's add a bullet point for consistency (and remove the extra space at the end of the line).

File: content/en/docs/v3.5/op-guide/configuration.md [hunk 2]
```diff
@@ -292,11 +290,9 @@ The security flags help to [build a secure etcd cluster][security].
 
 ### --peer-ca-file
 
-**DEPRECATED**
+**REMOVED**
 
-+ Path to the peer server TLS CA file. `--peer-ca-file ca.crt` could be replaced by `--peer-trusted-ca-file ca.crt --peer-client-cert-auth` and etcd will perform the same.
-+ default: ""
-+ env variable: ETCD_PEER_CA_FILE
+This flag has been removed. `--peer-trusted-ca-file ca.crt --peer-client-cert-auth` can be used to perform the same task as was performed by `--peer-ca-file ca.crt` previously.
```
Comment (nate-double-u) [inline content/en/docs/v3.5/op-guide/configuration.md:295]:
```suggestion
+ This flag has been removed. `--peer-trusted-ca-file ca.crt --peer-client-cert-auth` can be used to perform the same task as was performed by `--peer-ca-file ca.crt` previously.
```
Same here, add bullet for consistency, remove extra space at the end of the line.

**Review by nate-double-u — COMMENTED** [review]
A couple of formatting comments inline:

**Comment by nate-double-u** [comment]
Some of the other warnings state which version deprecation is going to happen in. ("WARNING: to be deprecated in v3.5.").

Should we add which version this was removed in? Was it v3.5? @spzala

**Review by spzala — APPROVED** [review]
Nice @Somoshree 
lgtm
@nate-double-u we can keep removed as done in this PR but they were deprecated since long v2.1 time so eventually we may want to simply remove them from this config file to keep it clean as possible?, but it's good for now.

**Comment by spzala** [comment]
> Some of the other warnings state which version deprecation is going to happen in. ("WARNING: to be deprecated in v3.5.").
> 
> Should we add which version this was removed in? Was it v3.5? @spzala

@nate-double-u good question, seems like they were removed in 3.4 but I think it's OK to me not be explicit about it. Thanks!

**Pull request merged**